src/app/clusters/microwave-oven-control-server/microwave-oven-control-server.cpp

```diff
@@ -209,7 +383,7 @@ void Instance::HandleAddMoreTime(HandlerContext & ctx, const Commands::AddMoreTi
     VerifyOrExit(opState != to_underlying(OperationalStateEnum::kError), status = Status::InvalidInState);
 
     // if the added cooking time is greater than the max cooking time, the cooking time stay unchanged.
-    VerifyOrExit(req.timeToAdd <= kMaxCookTime - GetCookTime(), status = Status::ConstraintError;
+    VerifyOrExit(req.timeToAdd <= mDelegate->GetMaxCookTime() - GetCookTime(), status = Status::ConstraintError;
```

Comment:
should we assert that GetCookTime >= MaxCookTime? is there an invariant guaranteeing this? given that the values are unsigned.